### DISCUSSION TITLE
installation blocked

### DISCUSSION THREAD

**Discussion by dabrack46** [FreeTubeApp/FreeTube#6109] [Q&A]
downloaded app easily.  Apple blocked installation  "Apple could not verify “FreeTube.app” is free of malware that may harm your Mac or compromise your privacy."
What should I do when Apple protects me so well?

**Comment by frrjr** [comment]
I'm getting the same issue

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/discussions/5260

see https://youtu.be/NOW5dyA_JgY
